conda_build/post.py

```diff
@@ -144,10 +144,10 @@ def compile_missing_pyc():
                            '-q', '-x', 'port_v3', sp_dir])
 
 
-def post_process(files, preserve_egg_dir=False):
+def post_process(files, preserve_egg_dir=False, pyc_compilation=True):
     remove_easy_install_pth(files, preserve_egg_dir=preserve_egg_dir)
     rm_py_along_so()
-    if config.CONDA_PY < 30:
+    if config.CONDA_PY < 30 and pyc_compilation:
```

Comment:
I don't know, but it probably has to with Python 3 stuff compiled files in a `__pycache__` folder

I haven't been able to use Python 3 much until recently, so now a lot of these Python 2/3 issues start creeping up on me :-)